docs/user-guide/custom.md

```diff
@@ -277,6 +278,21 @@ Note that, in the folder layout above, we are assuming that:
   }
   ```
 
+If you want to use an already exisiting python environment, you can use the parameter `environment_path` of your `model-settings.json`:
```

Comment:
maybe add in the docs that we can have envs etc. in the `enviornment_path`?

Reply:
Could you explain that further? I'm not quite sure what I'm supposed to do.

Reply:
@ramonpzg minor changes to the docs FYI